src/controllers/trustpub/github_configs/create/mod.rs

```diff
@@ -61,8 +61,14 @@ pub async fn create_trustpub_github_config(
         .load::<(i32, String, String, bool)>(&mut conn)
         .await?;
 
-    if !user_owners.iter().any(|owner| owner.0 == auth_user.id) {
-        return Err(bad_request("You are not an owner of this crate"));
+    let auth_owner = user_owners
```

Comment:
Nit: Instead of `auth_owner.3` It might be more readable to destructure it as a variable.
```suggestion
    let (.., verified) = user_owners
```
or
```suggestion
    let (_, _, _, verified) = user_owners
```